projects/Mallard/ios/Mallard/GuardianDailyDebug.entitlements

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
does this file need to exist now that `Mallard.entitlements` has been updated?

Reply:
it was generated by xcode. I wouldn't want to mess with it.